app/src/main/res/values-de/help.xml

```diff
@@ -0,0 +1,231 @@
+        <p>Falls du dein Wallet mit Schlüsseln wiederherstellen will!</p>
+        <p>Gib einen einzigartigen Walletnamen und Passwort ein. Das Passwort schützt deine Walletdaten auf dem Gerät.
+        Benutze ein starkes Passwort - noch besser ist eine Passphrase.<p>
+        <p>Gib deine Monero Adresse im Feld \"Öffentliche Adresse\" ein und fülle die Felder \"View Key\" und \"Spend Key\" aus.</p>
+        <p>Falls du die Blocknummer der ersten Transaktion dieser Adresse kennst gib sie in das Feld
+		\"Restore Height\" ein - dieses Feld leer zu lassen bedeutet die <em>gesamte</em> Blockchain nach
+        nach Transaktionen zu und von dieser Adresse zu durchsuchen. Das dauert <em>sehr lang</em>.</p>
+    ]]></string>
+
+    <string name="help_create_view"><![CDATA[
+        <h1>Wallet erstellen - View</h1>
+        <p>Falls du nur eingehende Transaktionen auf diese Wallet überwachen willst!</p>
+        <p>Gib einen einzigartigen Walletnamen und Passwort ein. Das Passwort schützt deine Walletdaten auf dem Gerät.
+        Benutze ein starkes Passwort - noch besser ist eine Passphrase.<p>
+        <p>Gib deine Monero Adresse im Feld \"Öffentliche Adresse\" ein und fülle das Feld \"View Key\" aus.</p>
+        <p>Falls du die Blocknummer der ersten Transaktion dieser Adresse kennst gib sie in das Feld
+		\"Restore Height\" ein - dieses Feld leer zu lassen bedeutet die <em>gesamte</em> Blockchain nach
+        nach Transaktionen zu und von dieser Adresse zu durchsuchen. Das dauert <em>sehr lang</em>.</p>
+    ]]></string>
+
+    <string name="help_details"><![CDATA[
+        <h1>Wallet Details</h1>
+        <h2>Öffentliche Adresse</h2>
+        Die öffentliche Adresse ist wie deine Kontonummer die du anderen Personen mitteilen kannst, ohne Angst zu haben dein Geld zu verlieren.
+	Andere Personen werden dir Monero an diese Adresse schicken.
+        <h2>Mnemonischer Seed</h2>
+        Dies sind die einzigen Daten die du benötigst um deine Wallet an einem späteren 
+	Zeitpunkt wiederherzustellen und vollen Zugriff auf deine Gelder zu erhalten.
+	Diese Daten geheim und sicher aufzubewahren ist sehr wichtig, da sie <em>jedem</em>
+        vollen Zugriff auf diese Gelder geben! Falls du ihn nicht an mehreren, sicheren Stellen aufgeschrieben hast mach dies bitte jetzt!
+        <h2>Walletdateien Wiederherstellungspasswort</h2>
+        Stelle sicher dieses Passwort aufzuschreiben. Falls du dein Gerät auf Werkseinstellungen zurücksetzt oder
+ 	die App deinstallierst brauchst du dieses Passwort um Zugriff auf dein Wallet zu erhalten.<br/>
+        <h3>CrAzYpass</h3>
+        Falls das hier angezeigte Passwort aus 52 alphanumerischen Zeichen in 4er Gruppen besteht - Glückwunsch!
+        Deine Walletdateien sind nun durch einen 256-Bit-Schlüssel gesichert, der durch Sicherheitsfeatures
+ 	deines Gerätes aus deiner Passwortphrase erstellt wurde (bei Erstellung oder Änderung der Passwortphrase).
+	Dies macht einen Hack der Walletdateien extrem schwer!<br/>
```

Comment:
"schwer" => "schwierig" :)

Reply:
Please tell me you are a native german speaker, otherwise I would feel ashamed ;)